src/soc/hw/noc/test/test_noc_buffer.py

```diff
@@ -0,0 +1,343 @@
+"""
```

Comment:
This file is missing a license header.

Reply:
We didn't add one for any of the other cocotb tests. Should I just use the same one we use for the verilog files?

Reply:
yes, please. Standard MIT license.

Reply:
`"""` is a documentation comment which will get parsed by Sphinx and similar tools. Since the license header shouldn't be seen by these tools make this a `#`-style comment please.